### PR TITLE
Add OPNsense support

### PR DESCRIPTION
**Pull request by axemann**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #8196

### CODE REVIEW

File: includes/polling/os/unix.inc.php [hunk 1]
```diff
@@ -159,4 +159,8 @@
     $output = preg_split("/ /", $device['sysDescr']);
     $version = $output[2];
     $hardware = $output[6];
+} elseif ($device['os'] == "opnsense") {
```
Comment (murrant) [inline includes/polling/os/unix.inc.php]: please move this to opnsense.inc.php instead of unix.inc.php

**Comment by murrant** [comment]
Thanks for the contribution. Mostly looks ok, the os polling is in the wrong file.

Also, SVG icons are preferred if you can get them: https://docs.librenms.org/Developing/os/Initial-Detection/#icon-and-logo

**Comment by axemann** [comment]
@murrant Glad to help out any way I can on an awesome project. :-)

I'll move the polling to a new file, but I'm not 100% sure how to revert my changes to unix.inc.php on here, outside of editing and re-pushing it.  I'll reach out to the OPNsense folks, too,  and see if they have the logo in SVG format.

**Comment by laf** [comment]
@axemann You can just remove the changes to unix.inc.php, commit that file and push it with the other changes and it will look like it's never happened :)

**Comment by axemann** [comment]
@laf Hopefully this last commit will do the trick...

**Pull request merged**